www/data_service/src/services/stream/stream.service.coffee

```diff
@@ -0,0 +1,36 @@
+class Stream extends Factory
+    constructor: ($log) ->
+        return class StreamInstance
```

Comment:
Here we returned a `class` instead of an initialized instance? I ask this question because the class is named as `StreamInstance` :)